src/osp/Satellites/SatActiveArea.cpp

```diff
@@ -40,14 +40,12 @@ using namespace Magnum::Math::Literals;
 
 using Magnum::Matrix4;
 
-namespace osp::universe
-{
+using namespace osp::universe;
```

Comment:
I recommend against using the "using namespace ..." syntax like this.

Better would be a namespace alias where you give the namespace a shorter name, but don't just import the entire namespace into the current scope.

e.g. 
namespace fbz = foo::bar::baz;

Reply:
This is quite a huge change to ask. `using namespace` is used in pretty much every source file. I've already touched almost every file in this PR, but I feel that this should go on a separate PR.

I'd argue that the use of `using namespace` across the sources are already quite restricted in most cases here:
* only used if a source file is part of the module the namespace implies
* only a few of the included files show up in the scope. Including the entire osp namespace would require manually including each individual files
* generic names aren't used anywhere and are mostly avoided

Reply:
Instead of "using namespace osp::universe", do "namespace uni = osp::universe", or something like it.

Don't need to change this for this PR. It can be done some other time.